### DISCUSSION TITLE
Does Artillery Support h264

### DISCUSSION THREAD

**Discussion by OmGuptaIND** [artilleryio/artillery#2484] [Artillery]
Great Work Guys, Was always looking for exactly what you do; thanks for saving me from writing my Scale Test Scripts.

The only issue I am currently facing is see. I want to load-test my website using Playwright, a video conferencing platform. I want the h264 codec to be supported. I know it's not free, and there is a workaround for using it in normal chromium cases. But for playwrights with Artillery as things are not normal you are running the code somewhere I will not be able to do the same things here.

Do you have any suggestions or workarounds I can try for it?

**Comment by hassy** [comment]
hi @OmGuptaIND 👋 I think you'd need to use your own Docker image as desribed here: https://github.com/artilleryio/artillery/discussions/2435
